IssueCreator/IssueCreator/Models/IssueToCreate.cs

```diff
@@ -0,0 +1,37 @@
+﻿using System;
+using System.Collections.Generic;
+
+namespace IssueCreator.Models
+{
+    internal class IssueToCreate : IEquatable<IssueToCreate>
+    {
+        public string Organization { get; set; }
+        public string Repository { get; set; }
+        public string AssignedTo { get; set; }
+        public string Title { get; set; }
+        public string Description { get; set; }
+        public List<string> Labels { get; set; }
+        public IssueDescription Epic { get; set; }
+        public IssueMilestone Milestone { get; set; }
+        public string Estimate { get; set; }
+        public bool CreateAsEpic { get; set; }
+
+        public bool Equals(IssueToCreate other)
```

Comment:
It doesn't warn about `GetHashCode`?

Reply:
No, because this is coming from the `IEquatable<IssueToCreate>` interface